data_steward/deid/config/ids/tables/procedure_occurrence.json

```diff
@@ -3,6 +3,10 @@
         {
```

Comment:
We said we would apply a cleaning rule further upstream. Please revert this change to `procedure_occurrence.json` when you get a chance.

Reply:
removed!